include/drm/drm_gem.h

```diff
@@ -119,6 +119,11 @@ struct drm_gem_object {
 	 * simply leave it as NULL.
 	 */
 	struct dma_buf_attachment *import_attach;
+
+	/*
+	 * is backed by scattered pages
+	 */
+	bool is_scattered;
```

Comment:
i want to keep this driver as upstreamable as possible, so modifying `drm_gem.h` seems like a giant no-no to me